test/spec/modules/conversantAnalyticsAdapter_spec.js

```diff
@@ -45,25 +48,33 @@ describe('Conversant analytics adapter tests', function() {
     };
     sandbox.stub(prebidGlobal, 'getGlobal').returns(getGlobalStub); // getGlobal does not seem to be available in testing so need to mock it
     clock = sandbox.useFakeTimers(DATESTAMP); // to use sinon fake timers they MUST be created before the interval/timeout is created in the code you are testing.
+
+    logInfoStub = sandbox.stub(utils, 'logInfo');/* .callsFake((arg, arg1, arg2) => { //debugging stuff
+      console.log(arg);
+      if (arg1) console.log(arg1);
+      if (arg2) console.log(arg2);
+    }); */
+
+    conversantAnalytics.enableAnalytics(VALID_ALWAYS_SAMPLE_CONFIG);
   });
 
   afterEach(function () {
     sandbox.restore();
     requests = []; // clean up any requests in our ajax request capture array.
+    conversantAnalytics.disableAnalytics();
```

Comment:
Should most testcases remove their own copy of `conversantAnalytics.disableAnalytics()` except for the ones that are testing cleanups?

Reply:
Yeah, it shouldn't be needed for test cases unless explicitly testing for it